src/main/java/org/pircbotx/Configuration.java

```diff
@@ -245,23 +245,23 @@ protected Configuration(Builder builder) {
 		/**
 		 * Enable or disable sending WEBIRC line on connect, default disabled
```

Comment:
Yes, that's what I mean't, and you need to do that with the rest of Configuration.java